src/bin/dhcp6/dhcp6_parser.yy

```diff
@@ -398,6 +400,7 @@ global_object: dhcp6_object
              | unknown_map_entry
              ;
 
+
 dhcp6_object: DHCP6 {
```

Comment:
Avoid this kind of spurious changes: it just makes the risk of conflict greater for no benefit (and BTW here I really prefer to have only one blank line).